docs/tox.ini

```diff
@@ -13,6 +13,7 @@ deps =
   sphinx==5.0.2
   sphinx-click==4.3.0
   sphinx_toolbox
+  sphinxcontrib-serializinghtml!=1.1.6
```

Comment:
Maybe pin <1.1.6? Seems more likely that there's a breaking change incompatible with our older sphinx than that it's just a bad release.